lightblue-ldap-integration-test/src/test/resources/crud/insert/department-insert-template.json

```diff
@@ -8,6 +8,6 @@
         "objectClass": ["top", "groupofNames"],
         "cn": "#cn",
         "description": "#description",
-        "member": [#members]
+        "member": ["#members"]
     }
```

Comment:
This change needs to be reverted. The test methods themselves are ensuring the quotation marks. I am fine if we want to change this, but lets do it in another issue/PR.